c172p.xml

```diff
@@ -1099,6 +1099,17 @@
             </product>
         </function>
 
+        <function name="fcs/aileron-pos-rad-avgd">
+            <description>Averaged aileron position</description>
+            <avg>
+                <property>fcs/left-aileron-pos-rad</property>
+                <product>
+                    <property>fcs/right-aileron-pos-rad</property>
```

Comment:
Did you try `<property>-fcs/right-aileron-pos-rad</property>` (without any spaces between - and the property)?

Reply:
Yes.
It gives 

> Fatal exception, '-' found in propertyname after 'jsbsim' name must begin with alpha Il or '_'

Reply:
It seems the minus sign is only supported for `<input>` in the summer, pure gain, etc. components. I'll merge the PR.

Reply:
I seemed to remember having seen this somewhere, but I couldn't find where.